doc/faq.rst

```diff
@@ -0,0 +1,191 @@
+
+The long answer answer is a bit more nuanced: maintaining Auto-sklearn requires a lot of time and
+effort, which would grow even larger when depending on more libraries. Also, adding more
+libraries would require us to generate meta-data more often. Lastly, having more choices does not
+guarantee a better performance for most users as having more choices demands a longer search for
+good models and can lead to more overfitting.
+
+Nevertheless, everyone can still add his or her favorite model to Auto-sklearn's search space by
+following the `examples on how to extend Auto-sklearn
+<https://automl.github.io/auto-sklearn/master/examples/index.html#extension-examples>`_.
+
+If there is interest in creating a auto-sklearn-contrib repository with 3rd-party models please
+open an issue for that.
+
+Can the preprocessing be disabled
+---------------------------------
+
+No, but we're working on that.
+
+Usage
+=====
+
+Only use interpretable models
+-----------------------------
+
+Auto-sklearn can be restricted to only use interpretable models and preprocessing algorithms.
+Please see the section :ref:`Restricting the searchspace` to learn how to restrict the models
+which are searched over or see
+`this example <examples/40_advanced/example_interpretable_models.html>`_.
+
+We don't provide a judgement which of the models are interpretable as this is very much up to the
+specific use case, but would like to note that decision trees and linear models usually most
+interpretable.
+
+Passing the number of models to try
+-----------------------------------
+
+In certain cases, for example for debugging, it can be helpful to limit the number of
+models to try. We do not provide this as an argument in the API as we believe that it
+should NOT be used in practice, but that the user should rather provide time limits.
+An example on how to add the number of models to try as an additional stopping condition
+can be found `in this github issue <https://github.com/automl/auto-sklearn/issues/451#issuecomment-376445607>`_.
+Please note that Auto-sklearn will stop when either the time limit or the number of
+models termination condition is reached.
+
+Ensemble contains only a dummy model
+------------------------------------
+
+TODO
+
+3. How to set useful budgets: https://github.com/automl/auto-sklearn/issues/57
+
+Parallel processing and oversubscription
+----------------------------------------
+
+TODO
+
+Meta-Learning
+=============
+
+Which datasets are used for meta-learning?
+------------------------------------------
+
+We updated the list of datasets used for meta-learning several times and this list now differes
+significantly from the original 140 datasets we used in 2015 when the paper and the package were
+released. An up-to-date list of `OpenML task IDs <https://docs.openml.org/#tasks>`_ can be found
+on `github <https://github.com/automl/auto-sklearn/blob/master/scripts/update_metadata_util.py>`_
+
+Which meta-features are used for meta-learning?
```

Comment:
Add section on "How can I exclude datasets from the meta-learning?", e.g. if I want to run auto-sklearn on these